fedimint-core/src/config.rs

```diff
@@ -269,10 +238,33 @@ impl<M> Default for ModuleGenRegistry<M> {
     }
 }
 
+/// Module **generation** (so passed to dkg, not to the module itself) config
+/// parameters
+#[derive(Debug, Clone, Default)]
+pub struct ConfigGenParams(serde_json::Value);
```

Comment:
As things were currently, the use of this type was wierd, where the keys were used for module kind instead of a keys to multiple values.

We'll have to change how the details here work, but currently I'm only interested in an ability to inject config gen params for a given module kind from the outside to unblock 3rd party out-of-tree building.

@elsirion 
Possibly we should have a "ServerModuleGen::default_params() -> ConfigGenParams` and just allow overrides from the CLI in the form of `KIND.some.js.key = value` and in the UI, just display the `ConfigGenParams` whole, as a serialized json, and allow user to edit them if they wish. Seems like a simple but functional way to handling it all.

Reply:
> just display the `ConfigGenParams` whole, as a serialized json, and allow user to edit them if they wish

Yes I think that will be how we should handle it in the UI/API during config setup.  The DKG CLI will eventually just be making API calls, but could support downloading the current config, manually editing, and then uploading.